src/config.js

```diff
@@ -89,7 +89,12 @@ const context = {
 	GITHUB_REPOSITORY: getVar({
 		key: 'GITHUB_REPOSITORY',
 		required: true
-	})
+	}),
+	SKIP_PR: getVar({
+		key: 'SKIP_PR',
+		required: false,
```

Comment:
required defaults to false, so you can remove it here.

Reply:
Updated